packages/eslint-plugin-template/tests/rules/headings-consecutive-order/cases.ts

```diff
@@ -0,0 +1,53 @@
+import { convertAnnotatedSourceToFailureCase } from '@angular-eslint/utils';
+import { MessageId } from '../../../src/rules/headings-consecutive-order';
+
+const messageId: MessageId = 'headingsOrder';
+
+export const valid = [
+  // valid examples from https://www.w3.org/WAI/tutorials/page-structure/headings/
+  {
+    // organize passages on text
+    code: `<h1>Heading 1</h1>
+            <h2>Heading 2</h2>
+            <h2>Heading 2</h2>
+            <h2>Heading 2</h2>`,
```

Comment:
I wonder if it would be worthwhile to ignore heading elements inside an `<ng-template>` since templates are rendered separately and not typically in the order they are written in the code.

Reply:
Thanks, yes, technically it is not expected to follow the ordering..